evals/registry/evals/audio_tasks.yaml

```diff
@@ -125,26 +160,65 @@ transcript-ser.validation.v1:
     dataset: hf://Zahra99/IEMOCAP_Audio?split=session5
     text_only: true
 
-transcript-sqa:
-  id: transcript-sqa.validation.v1
+transcript-sqa-heysquad:
```

Comment:
I'm starting to feel like we need tests for yaml files.
The basic ones would be: make sure it can be loaded and that if it start with `transcript`, then `text_only` must be True. 

Another one might be to see if it can find matching `audio-x` and `transcript-x` and make sure everything matches except for `text_only`.

Not for this PR though.

Reply:
Possibly. Another option might be to just load a known model and run a few samples from all evals through it and validate that everything still works, more comprehensive than just loading the YAML. Agree either is out of scope for this PR though.